app/1.0.x/upgrading.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Upgrading guide
+---
+
+## Introduction
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> What follows is the upgrade guide for 1.0.x.
+  If you are trying to upgrade to an earlier version of Kong, please read
+  <a href="https://github.com/Kong/kong/blob/master/UPGRADE.md">UPGRADE.md file in the Kong repo</a>
+</div>
+
+Unless indicated otherwise in one of the upgrade paths of this document, it is
```

Comment:
> one of the upgrade paths of this document

This paragraph only makes sense in UPGRADE.md.

This section is not really an Introduction, it is really about upgrading patch releases (with no migrations). I would take everything from "Assuming that..." up to "...in-flight connections") and move it to a "Upgrading patch releases" at the bottom (with a new introductory paragraph explaining what patch releases are, i.e. "1.0.x to 1.0.y")

Reply:
I decided to just remove this part. We cal always add it later when we do a point release. And it is present in the `UPGRADING.md` file anyway.

Reply:
> We cal always add it later when we do a point release

I bet we _will_ forget to do it when we do a point release ;) At the very least please add a sentence about 1.0.x point releases pointing to `UPGRADE.md`, in order to future-proof this page.

Reply:
Ok. I ended up adding the whole thing in a section. It was faster to do than coming with a new section and then pointing to `UPGRADING.md`

Reply:
I think this sentence is still misplaced: "Unless indicated otherwise in one of the upgrade paths of this document", because no sections in this document indicate so. It is very much an artifact of `UPGRADE.md`.